File: codingsample/202008/20200824/Rains_of_Reason_4_alphabeticShift.py
def alphabeticShift(inputString):
    temp = list(inputString)
    for i in range(len(temp)):
        if temp[i] == 'z': temp[i]='a'
        else: temp[i] = chr(ord(temp[i])+1)
    inputString = "".join(temp)
    return inputString

inputs = "crazy"
print(alphabeticShift(inputs))

File: codingsample/202008/20200824/Rains_of_Reason_4_alphabeticShift.c
#include <stdio.h>
#include <stdlib.h>
#include <stdbool.h>
#include <string.h> 

char * alphabeticShift(char * inputString) {
    int len = strlen(inputString),i;
    for(i=0;i<len;i++) {
        if(inputString[i]=='z') inputString[i]='a';
        else inputString[i] = inputString[i]+1;
    }
    return inputString;
}



int main() {
	char* input = "crazy";
	alphabeticShift(input);
	printf("%s\n",input);
}


